tests/model_specs/bounce_token_tests.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+require('../bootstrap');
+var Model = require('../../lib');
+var expect = require('chai').expect;
+var mongoose = Model._mongoose;
+var config = require('config');
+var dbUri = config.get('Hoist.mongo.db');
+describe('BouncerToken', function () {
+  before(function (done) {
+    if (mongoose.connection.db) {
+      return done();
+    }
+    mongoose.connect(dbUri, done);
+  });
+  after(function (done) {
+    Model.BouncerToken.remove({}, function () {
+      mongoose.disconnect(function () {
+        delete mongoose.connection.db;
+        done();
+      });
+    });
+  });
+  describe('save token with display properties', function () {
+    var saved, token;
+    before(function () {
+      token = new Model.BouncerToken({
+        application: 'appid',
+        environment: 'live',
+        displayProperties: [
+          {
+            name: 'Application Name',
+            value: 'Hoist Apps Limited'
+          }
+        ]
+      });
+      return (saved = token.saveAsync());
+    });
+    it('should save correctly', function () {
+      return expect(saved).to.be.fulfilled;
+    });
+    it('should save display properties', function () {
+      return expect(token.displayProperties[0]).to.have.property('_id')
```

Comment:
does each display prop actually need an _id? 
set _id to false in the schema and there wont be an id on each display prop
http://mongoosejs.com/docs/api.html#schema_Schema